libr/cons/input.c

```diff
@@ -166,9 +167,18 @@ R_API int r_cons_arrow_to_hjkl(int ch) {
 						p = 0;
 					}	
 				} while (ch != 'M' && ch != 'm');
+				// right click
+				if (atoi (vel) == 2) {
```

Comment:
we can probably compute atoi(vel) once and make a switch statement below instead of atoiing 3 times  the same thing